app/code/Magento/Backup/Test/Unit/Model/BackupFactoryTest.php

```diff
@@ -113,6 +113,13 @@ public function testCreate()
         )->will(
             $this->returnSelf()
         );
+        $this->_backupModel->expects(
+            $this->once()
```

Comment:
Please reformat similar to change in `app/code/Magento/Backup/Model/BackupFactory.php`, current ugly formatting is derived from poor formatting tool.